doc/cmp.txt

```diff
@@ -679,6 +679,13 @@ view.docs.auto_open~
 
   Specify whether to show the docs_view when selecting an item.
 
+                                                 *cmp-config.view.follow_cursor*
+view.follow_cursor*
```

Comment:
Hello, reading the rest of the final change it seems the help tags here are wrong, they should use `view.entries.follow_cursor` since the option is under `entries` 🤔

Reply:
Good spot, thanks! I'll submit a fix today